caculate percent commission/main.js

```diff
@@ -0,0 +1,13 @@
+calPerCommission = (sales) => {
+    var sales = prompt('Nhập doanh số bán được: ')
+    if (sales <= 100000000) {
+        alert('Số hoa hồng bạn nhận được là: ' + sales*0.05)
+    }
+    else if (sales > 100000000 && sales <= 300000000) {
```

Comment:
cách đặt dấu `}` và `else if` phải là:
```suggestion
    } else if (sales <= 300000000) {
```
Bên trên đã kiểm tra `sales <= 100000000` rồi thì xuống dưới này làm gì có chuyện `sales <= 100000000` nữa mà phải kiểm tra `sales > 100000000` tương tự cho các logic bên dưới